docs/design/values.md

```diff
@@ -127,22 +157,24 @@ Value bindings require the matched expression to be a _value expression_,
 converting it into one as necessary.
 
 A _variable pattern_ can be introduced with the `var` keyword to create an
-object with storage when match.d Every binding pattern name introduced within a
+object with storage when matched. Every binding pattern name introduced within a
 variable pattern is called a _variable binding_ and forms a
-[_reference expression_](#reference-expressions) to an object within the
-variable pattern's storage when used. Variable patterns require their matched
-expression to be an _initializing expression_ and provide their storage to it to
-be initialized.
+[_durable reference expression_](#durable-reference-expressions) to an object
+within the variable pattern's storage when used. Variable patterns require their
+matched expression to be an _initializing expression_ and provide their storage
+to it to be initialized.
 
 ```carbon
 fn MutateThing(ptr: i64*);
 
 fn Example() {
-  // Both `1` and `2` here start as value expressions. That is a match for `1`.
-  // For `2`, the variable binding requires it to be converted to an
-  // initializing expression by using the value `2` to initialize the provided
-  // variable storage that `y` will refer to.
-  let (x: i64, var y: i64) = (1, 2);
+  // `1` starts as a value expression, and the `let` binds a name to it.
```

Comment:
This phrasing feels awkward to me, because I think of `let` as binding a name to a value, not an expression.

```suggestion
  // `1` starts as a value expression, which is what a `let` binding expects.
```